tests/integration/test_eoc_journal.py

```diff
@@ -111,7 +157,78 @@ def test_pb_answer_blocks_listed_in_edit_view(self):
             label = item.find_element_by_css_selector('label')
             self.assertEqual(label.text, title)
 
+    def test_pb_answers_listed_in_student_view(self):
+        self.patch(
+            'eoc_journal.eoc_journal.EOCJournalXBlock.selected_pb_answer_blocks',
+            default_pb_answer_block_ids
+        )
+        self.set_standard_scenario()
+        self.go_to_view('student_view')
+        self.fix_js_environment()
+
+        element = self.get_element_for_selected_pb_answers()
+
+        items = element.find_elements_by_css_selector('.eoc-selected-answers-section')
+        self.assertEqual(len(items), 1)
+
+    def test_pb_answers_rendered_in_student_view(self):
```

Comment:
I don't think this test is very useful on its own, I think it would make sense to combine it with the test above.